src/main/kotlin/app/revanced/patches/youtube/misc/hdrbrightness/annotations/HDRBrightnessCompatibility.kt

```diff
@@ -5,7 +5,7 @@ import app.revanced.patcher.annotation.Package
 
 @Compatibility(
     [Package(
-        "com.google.android.youtube", arrayOf("17.24.34", "17.24.35", "17.25.34", "17.26.35")
+        "com.google.android.youtube", arrayOf("17.25.34")
```

Comment:
Why did you remove those versions here?

Reply:
Always aim for the latest supported version from the docs.

Reply:
havn't tested those versions, so i cannot tell if the patch is compatible

Reply:
> havn't tested those versions, so i cannot tell if the patch is compatible

only test the latest one then

Reply:
tested and updated 👍🏻